src/InfRcTransport.h

```diff
@@ -49,6 +49,29 @@ namespace RAMCloud {
 
 /**
  * Transport mechanism that uses Infiniband reliable queue pairs.
+ *
+ * NOTE: If you choose to use this transport for RAMCloud server, RAMCloud coordinator,
+ * and any of your RAMCloud client libraries, you will need the following environment
+ * variables set:
+ *
+ * - RDMA_DEV
+ * - RDMA_DEVPORT
+ * - RDMA_GID_INDEX
+ *
+ * Depending on which Infiniband or compatible device you are using, and the
```

Comment:
Really should be `Depending on which RDMA device you are using`.  Infiniband doesn't have GID indexes as I recall (the whole `ah` structure is RoCE specific if I understand things correctly).

This patch actually breaks infiniband (not that we are concerned about that here).

Reply:
Oh, didn't realize that ah is RoCE specific. Ok, will add in a RoCE toggle accordingly.

Reply:
Added in comment to the class stating it is modified for RoCEv2 & won't work otherwise. We can revisit if InfRcTransport is needed for something non-ROCE